cloud/logger/src/EmptyLogger.ts

```diff
@@ -0,0 +1,19 @@
+import {Logger} from './Logger';
+
+export class EmptyLogger implements Logger {
```

Comment:
Yes, you can use classes as interfaces, so you mean turning the `Logger` interface itself into the noop-logger class right? :+1: